ur_robot_driver/CMakeLists.txt

```diff
@@ -88,17 +89,24 @@ add_library(ur_robot_driver_plugin
 target_link_libraries(ur_robot_driver_plugin ur_client_library::urcl ${catkin_LIBRARIES})
 add_dependencies(ur_robot_driver_plugin ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 
+add_library(urcl_log_handler
+  src/urcl_log_handler.cpp
+)
+target_link_libraries(urcl_log_handler ${catkin_LIBRARIES} ur_client_library::urcl)
+
 add_executable(ur_robot_driver_node
   src/dashboard_client_ros.cpp
   src/hardware_interface.cpp
   src/hardware_interface_node.cpp
+  src/urcl_log_handler.cpp
```

Comment:
Alternatively we could link the driver node to the log handler library. I don't have any strong opinion on that, though.

Reply:
I will give it a look.